### PR TITLE
fix(examples): fix in webpack3 examples

### LINKED ISSUE #2559: Example #3 is not running
Hello everyone,

I using the webpack-Example "3".
When I use the command "npm start" then the following error occured: 
_"Error: babel-plugin-direct-import: contains unknown keys { modules }"_

I have the start.js or other script not touched!

Can everyone help me?

many thanks in advance

Sven

### PR DESCRIPTION
**Pull request by layershifter**
Fixes #2559.

### CODE REVIEW

File: examples/webpack3/.babelrc [hunk 1]
```diff
@@ -15,13 +15,7 @@
...
     "@babel/plugin-proposal-class-properties",
     "@babel/plugin-proposal-export-default-from",
-    ["direct-import", { "modules": [
-      "semantic-ui-react", {
-        "name": "semantic-ui-react",
-        "indexFile": "semantic-ui-react/dist/es/index.js"
-      }
-    ]}],
-    "lodash"
+    ["lodash", {"id": ["lodash", "semantic-ui-react"]}]
```
Comment (layershifter) [inline examples/webpack3/.babelrc:18]: `babel-plugin-direct-import` seems doesn't work at all :disappointed: Replaced with `babel-plugin-lodash`

File: examples/webpack3/package.json [hunk 2]
```diff
@@ -7,34 +7,39 @@
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
     "semantic-ui-less": "^2.2.12",
-    "semantic-ui-react": "^0.77.0"
+    "semantic-ui-react": "^0.78.2"
```
Comment (layershifter) [inline examples/webpack3/package.json:10]: Updated to latest SUIR :+1:

File: examples/webpack3/package.json [hunk 3]
```diff
@@ -7,34 +7,39 @@
...
+    "postcss-loader": "^2.1.0",
+    "react-dev-utils": "^5.0.0",
+    "style-loader": "^0.20.2",
+    "uglifyjs-webpack-plugin": "^1.2.0",
+    "url-loader": "^0.6.2",
+    "webpack": "^3.11.0",
+    "webpack-bundle-analyzer": "^2.10.0",
+    "webpack-dev-server": "^2.11.1",
+    "webpack-manifest-plugin": "^1.3.2",
+    "webpack-merge": "^4.1.1"
```
Comment (layershifter) [inline examples/webpack3/package.json:42]: Added missing deps and updated existing

File: examples/webpack3/src/components/Navbar/NavbarChildren.less [hunk 4]
```diff
@@ -1,5 +1,5 @@
 @import '~heading.less';
 
 .container {
-  margin-top: @relative64px;
+  padding-top: @relative64px;
```
Comment (layershifter) [inline examples/webpack3/src/components/Navbar/NavbarChildren.less:4]: Wrong `top` there was :star:

**Comment by layershifter** [comment]
Tested locally, everything works as expected :+1: Merged

**Comment by layershifter** [comment]
Tested locally, everything works as expected :+1: Merged

**Comment by progrookie** [comment]
Thanks for the fix!!!

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.78.3`.

**Pull request merged**